--- CMakeLists.txt.boost	2011-03-17 11:30:20.000000000 +0100
+++ CMakeLists.txt	2011-03-17 11:30:43.000000000 +0100
@@ -171,6 +171,7 @@
 	    IF( Boost_FOUND )
 		    link_directories( ${Boost_LIB_DIR} )
 		    INCLUDE_DIRECTORIES( ${Boost_INCLUDE_DIR} )
+		    ADD_DEFINITIONS(-DBOOST_FILESYSTEM_VERSION=2)
 		    SET( SL_BOOST_LIBS boost_system-mt )
 	    ELSE()
 		    MESSAGE( FATAL_ERROR "boost library not found! Please install the package or toggle OPTION_TORRENT_SYSTEM to OFF")
